scrapy/core/downloader/handlers/http11.py

```diff
@@ -366,13 +366,13 @@ def _cb_bodyready(self, txresponse, request):
         fail_on_dataloss = request.meta.get('download_fail_on_dataloss', self._fail_on_dataloss)
 
         if maxsize and expected_size > maxsize:
-            error_msg = ("Cancelling download of %(url)s: expected response "
-                         "size (%(size)s) larger than download max size (%(maxsize)s).")
-            error_args = {'url': request.url, 'size': expected_size, 'maxsize': maxsize}
+            warning_msg = ("Expected response size (%(size)s) larger than "
+                           "download max size (%(maxsize)s) in request %(request)s.")
+            warning_args = {'request': request, 'size': expected_size, 'maxsize': maxsize}
+
+            logger.warning(warning_msg, warning_args)
 
-            logger.error(error_msg, error_args)
             txresponse._transport._producer.loseConnection()
-            raise defer.CancelledError(error_msg % error_args)
```

Comment:
I’m not sure removing this line was part of the #3874, where the reported only complaints about the logging level, not that the request is sent to the errback, which is probably the expected behavior.

Reply:
Yeah, you are right, I returned it back, thanks.